api/db/changelog/db.changelog-master.xml

```diff
@@ -67,4 +67,5 @@
   <include file="changelog/db.changelog-57-add-era-commons-entries.xml"/>
   <include file="changelog/db.changelog-58-update-access-modules.xml"/>
   <include file="changelog/db.changelog-59-cluster-config-default.xml"/>
+  <include file="changelog/db.changelog-60-drop-training-completion-time-col.xml"/>
```

Comment:
I think you want to include changelog 61 here as well. (Also, the backfill has gone in since this, so you will want to rebase this)